### PR TITLE
Add licence to project

### PR DESCRIPTION
**Pull request by nowak-pawel-ds**

### CODE REVIEW

File: core/src/main/resources/apps/bulma/components/box/box.html [hunk 1]
```diff
@@ -1,3 +1,21 @@
+<!--
```
Comment (paweljozwicki) [inline core/src/main/resources/apps/bulma/components/box/box.html]: Please use 

This will remove this comment from the created site

Comment (nowak-pawel-ds) [reply core/src/main/resources/apps/bulma/components/box/box.html]: Done

Comment (paweljozwicki) [reply core/src/main/resources/apps/bulma/components/box/box.html]: I think few left: column, levelitem, positionedlevel

Comment (nowak-pawel-ds) [reply core/src/main/resources/apps/bulma/components/box/box.html]: Done

**Review by paweljozwicki — APPROVED** [review]

**Pull request merged**